{{cookiecutter.app_name}}/tests/test_database.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+"""Database unit tests."""
+import pytest
+from flask_login import UserMixin
+
+from {{cookiecutter.app_name}}.database import Column, PkModel, db
+
+
+class ExampleUserModel(UserMixin, PkModel):
+    """Example model class for a user."""
+
+    __tablename__ = "testusers"
+    username = Column(db.String(80), unique=True, nullable=False)
+    email = Column(db.String(80), unique=True, nullable=False)
+
+    def __init__(self, username, email):
+        """Create instance."""
+        super().__init__(username=username, email=email)
+
+
+@pytest.mark.usefixtures("db")
+class TestCRUDMixin:
+    """CRUDMixin tests."""
+
+    def test_create(self):
+        """Test CRUD create."""
+        user = ExampleUserModel.create(username="foo", email="foo@bar.com")
+        assert ExampleUserModel.get_by_id(user.id).username == "foo"
+
+    def test_delete_with_commit(self):
+        """Test CRUD delete with commit."""
+        user = ExampleUserModel("foo", "foo@bar.com")
+        user.save()
+        assert ExampleUserModel.get_by_id(user.id) is not None
```

Comment:
Would you be able to split this test into two so that `assert ExampleUserModel.get_by_id(user.id) is not None` is in its own test case?

Where possible, I think it's best to follow the arrange-act-assert pattern in unit tests where the order of "acts" and "asserts" is not mixed.